vpr/src/pack/post_routing_pb_pin_fixup.cpp

```diff
@@ -0,0 +1,1090 @@
+         * nothing to be changed. Move on to the next net.
+         */
+        if (routing_net_id == cluster_net_id) {
+            continue;
+        }
+
+        /* Update the clustering context with net modification */
+        clustering_ctx.post_routing_clb_pin_nets[blk_id][pb_graph_pin->pin_count_in_cluster] = routing_net_id;
+
+        std::string routing_net_name("unmapped");
+        if (clustering_ctx.clb_nlist.valid_net_id(routing_net_id)) {
+            routing_net_name = clustering_ctx.clb_nlist.net_name(routing_net_id);
+        }
+
+        std::string cluster_net_name("unmapped");
+        if (clustering_ctx.clb_nlist.valid_net_id(cluster_net_id)) {
+            cluster_net_name = clustering_ctx.clb_nlist.net_name(cluster_net_id);
+        }
+
+        VTR_LOGV(verbose,
+                 "Fixed up net '%s' mapping mismatch at clustered block '%s' pin 'grid[%ld][%ld].%s.%s[%d]' (was net '%s')\n",
+                 routing_net_name.c_str(),
+                 clustering_ctx.clb_nlist.block_pb(blk_id)->name,
+                 grid_coord.x(), grid_coord.y(),
+                 clustering_ctx.clb_nlist.block_pb(blk_id)->pb_graph_node->pb_type->name,
+                 get_pb_graph_node_pin_from_block_pin(blk_id, physical_pin)->port->name,
+                 get_pb_graph_node_pin_from_block_pin(blk_id, physical_pin)->pin_number,
+                 cluster_net_name.c_str());
+
+        /* Update counter */
+        num_mismatches++;
+    }
+}
+
+/********************************************************************
+ * Find an unused pb_route from the other pins in this port 
+ * The pb_route should be remapped to an invalid net, becoming unused
+ * at post routing stage.
+ *
+ * This function will return the first one we can find.
+ *******************************************************************/
+static int find_target_pb_route_from_equivalent_pins(const AtomContext& atom_ctx,
+                                                     const ClusteringContext& clustering_ctx,
+                                                     const ClusterBlockId& blk_id,
+                                                     t_pb* pb,
+                                                     const t_pb_graph_pin* source_pb_graph_pin,
+                                                     const AtomNetId& target_net,
+                                                     const bool& verbose) {
+    VTR_ASSERT(source_pb_graph_pin->parent_node->is_root());
+
+    std::vector<int> pb_route_indices;
+
+    for (int pb_type_pin = 0; pb_type_pin < pb->pb_graph_node->pb_type->num_pins; ++pb_type_pin) {
+        const t_pb_graph_pin* pb_graph_pin = get_pb_graph_node_pin_from_block_pin(blk_id, pb_type_pin);
+
+        if (PortEquivalence::FULL != pb_graph_pin->port->equivalent) {
+            continue;
+        }
+
+        /* Limitation: bypass output pins now
+         * TODO: This is due to the 'instance' equivalence port 
```

Comment:
Please add a GitHub issue for this and put the issue number in the TODO comment.

Reply:
Created a pull request at https://github.com/verilog-to-routing/vtr-verilog-to-routing/issues/1578